lib/SILOptimizer/SemanticARC/SemanticARCOptVisitor.h

```diff
@@ -53,9 +53,9 @@ struct LLVM_LIBRARY_VISIBILITY SemanticARCOptVisitor
       : ctx(fn, onlyGuaranteedOpts,
             InstModCallbacks(
                 [this](SILInstruction *inst) { eraseInstruction(inst); },
-                [](SILInstruction *) {}, [](SILValue, SILValue) {},
-                [this](SingleValueInstruction *i, SILValue value) {
-                  eraseAndRAUWSingleValueInstruction(i, value);
+                [this](Operand *use, SILValue newValue) {
+                  use->set(newValue);
```

Comment:
This hunk may be slightly confusing. What is happening is that rather than passing an empty argument for createdNewInst, I am just using an overload of InstModCallback's constructor that omits it (and thus assumes we want the default... which we do).